src/website/app/demos/Link/index.js

```diff
@@ -16,14 +16,23 @@
 
 /* @flow */
 import { componentTheme } from '../../../../Link/Link';
+import bestPractices from './bestPractices';
 import examples from './examples';
 
 const doc = require('!!react-docgen-loader!../../../../Link/Link');
 
 export default {
+  bestPractices,
   componentTheme,
   doc,
   examples,
   slug: 'link',
-  title: 'Link'
+  title: 'Link',
+  whenHowToUse: `Links should be used when the browser location will change.
+Links should be used inline and be clear about where the user will navigate.
+Avoid using "click here" or similar for Link content.
+
+Generally, for stand-alone navigation actions, use a [Button](./button).
```

Comment:
I think we need a more nuanced take here. Stand-alone navigation is still navigation not an action. They should use a Link.

Reply:
Definitely agreed that they should use a Link. We don't currently provide an out of the box solution for a Link that looks like a Button, but it's on the roadmap. Should we just point out here that they'll need to style the Link themselves if they want a call to action that links to another page?

Reply:
Nah, I'd rather not draw attention to a (temporary) lack of our library. I also don't think those things (standalone links and links that look like buttons) have to be tied together. Maybe rather than being specific about it, we can just be more vague by removing references to "use inline"? The examples make it clear that it can be used both within prose and standalone.